src/libraries/System.Private.CoreLib/src/System/Globalization/NumberFormatInfo.cs

```diff
@@ -55,6 +61,21 @@ public sealed class NumberFormatInfo : IFormatProvider, ICloneable
         internal string _percentSymbol = "%";
         internal string _perMilleSymbol = "\u2030";
 
+        internal byte[]? _positiveSignUtf8;
```

Comment:
Sorry, I'm not following. What are you suggesting caching that's not already being cached here?

Reply:
Avoiding the call to `Encoding.UTF8GetBytes` entirely for the most common scenario, since that won't be constant foldable as its currently setup and would be needed per symbol.

It might not actually be beneficial in practice, just a scenario that came to mind.

Reply:
> Avoiding the call to Encoding.UTF8GetBytes entirely for the most common scenario, since that won't be constant foldable as its currently setup and would be needed per symbol.

Oh, I see, you mean initializing the field so that we don't pay for the GetBytes on the first use, basically just hardcoding it to the UTF8-equivalent of the literal the UTF16 fields are initialized to?

Reply:
Right. I imagine we could optimize both the default and invariant cases this way and that it might minimally help things

Reply:
Yeah, that makes sense.  Let me see how hard it would be.

Reply:
I'm going to defer for now.  There are several places these fields are set, and I'm not even sure if the initial const string values initializing the utf16 fields are ever actually used... they seem to get overwritten by the other code paths.  I agree there's probably some value in doing something like what you suggest, in particular to avoid duplicate byte[]s being created for different cultures that actually share a common value for some of these fields, but let's hold off on complicating things for now to reap those benefits.